packages/synapse-react-client/src/components/SetBasicAccessRequirementFields/SetBasicAccessRequirementFields.test.tsx

```diff
@@ -91,31 +88,39 @@ function setUp(props: SetBasicAccessRequirementFieldsProps) {
 describe('SetBasicAccessRequirementFields', () => {
   beforeEach(() => jest.clearAllMocks())
   beforeAll(() => server.listen())
-  afterEach(() => server.restoreHandlers())
+  afterEach(() => server.resetHandlers())
```

Comment:
[`resetHandlers`](https://mswjs.io/docs/api/setup-server/reset-handlers/) is necessary to remove all runtime request handlers